go/integration_test/interpreter/dynamic_gas.go

```diff
@@ -839,7 +845,12 @@ func gasDynamicSelfDestruct(revision Revision) []*DynGasTest {
 				return tosca.Value{}
 			})
 
-			mock.EXPECT().AccountExists(targetAddress).AnyTimes().Return(!empty)
+			nonce := uint64(0)
+			if !empty {
+				nonce = 1
```

Comment:
same here, should it not set codeSize as well ?

Reply:
It is enough if one of the three properties (nonce, balance, code) is non-zero to make an account empty. And for simplicity, the code here just sets to nonce to 1 to make the account non-empty.

Reply:
ahhh right.